data/parsers/okinawa.py

```diff
@@ -0,0 +1,43 @@
+import sys
+import requests
+import csv
+import io
+from datetime import datetime
+
+from .utils import store_data, stoi
+
+# ------------------------------------------------------------------------
+# Globals
+
+#URL  = "https://github.com/spigolotti/covid2019data_okinawa/okinawaCOVID19.csv"
+URL = "https://github.com/spigolotti/covid2019data_okinawa/blob/master/okinawaCOVID19.csv?raw=true"
+cols = ['time', 'cases', 'deaths', 'hospitalized', 'icu', 'recovered']
+
+# ------------------------------------------------------------------------
+# Main point of entry
+
+def parse():
+    r  = requests.get(URL)
+    if not r.ok:
+        print(f"Failed to fetch {URL}", file=sys.stderr)
+        exit(1)
+        r.close()
+#    regions = dict(JPN-Okinawa=[])
```

Comment:
`regions = {'JPN-Okinawa'=[]}` would have been the right way ;) Can you remove the commented out old code in this parser?

Reply:
removed empty line, added the ICU source and removed the commented old code